src/App.tsx

```diff
@@ -134,22 +134,17 @@ const isComplianceDeclined = localStorage.getItem(`${Config.cortexApi.scope}_Com
 
 const complianceSupportModal = showCompliance && !isComplianceAccepted && !isComplianceDeclined ? <ComplianceSupportModal onAcceptDataPolicy={handleAcceptDataPolicy} /> : '';
 
-const VersionContainer = (props) => {
-  const { appVersion, componentsVersion } = props;
-  return (
-    <div className="version" style={{ display: 'none' }}>
-      <span>{ `${intl.get('app-version')}: ${appVersion}` }</span>
-      <span>{ `${intl.get('components-version')}: ${componentsVersion}` }</span>
```

Comment:
We still have a components version available under `/src/components/package.json`.  Are we scrapping that?  We may need to change the github prompts.  Also how will we keep track with the components up in NPM?  Or are we getting rid of the components that should be up there entirely?

Reply:
Agree with the above point. We still need to surface the versions of those components in the dom.

Reply:
Let's discuss this further tomorrow.

Reply:
reverted